packages/workflow/src/errors/execution-cancelled.error.ts

```diff
@@ -1,10 +1,12 @@
 import { ExecutionBaseError } from './abstract/execution-base.error';
 
+export type ExecutionCancellationReason = 'timeout' | 'manual' | 'system' | 'unknown';
```

Comment:
Could we have a JSDoc comment for this type that would explain the different values?

Reply:
Also, is the system used anywhere else besides when shutting down? Should be `system-shutdown` instead?